README.md

```diff
@@ -1,20 +1,20 @@
 # NI JavaScript and TypeScript Style Guide
 
-![Logo](resources/logo.svg)
+![Logo](https://raw.githubusercontent.com/ni/javascript-styleguide/HEAD/resources/logo.svg)
```

Comment:
Update logo usage in README to point to github served content to make visilbe on npmjs.com README

Reply:
Understood!